doc/Native APIs.md

```diff
@@ -0,0 +1,417 @@
+Native API Documentation
+========================
+
+This documents the APIs that can be used to implement native wrappers. There are
+three types of APIs:
+
+- `student_view_data`: Exposes block settings and content as JSON data. Can be
+  retrieved via the edX Course Blocks API. Does not include user-specific data
+  (i.e. items derived from `Scope.user_state` fields).
```

Comment:
@mtyaka - I just tried again, and now it's working for me with just `requested_fields=student_view_data` -- not quite sure why it wasn't before?